server/controllers/transferwise.ts

```diff
@@ -50,14 +50,27 @@ export async function payBatch(
     if (!remoteUser.isAdmin(host.id)) {
       throw new errors.Unauthorized('User must be admin of host collective');
     }
-    // TODO: ensure expense ids are sent with the OTT retry request
     const expenseIds = body?.expenseIds?.map(id => idDecode(id, IDENTIFIER_TYPES.EXPENSE));
     const expenses = await models.Expense.findAll({
       where: { id: { [Op.in]: expenseIds } },
-      include: [{ model: models.PayoutMethod, as: 'PayoutMethod' }],
+      include: [
+        { model: models.PayoutMethod, as: 'PayoutMethod', required: true },
+        {
+          model: models.Collective,
```

Comment:
I don't know whether we're later using `expense.collective`, but if the goal here is just to add an `INNER JOIN` to filter expenses then we can easily save some RAM (which should help if we're trying to address a timeout issue)

```suggestion
          model: models.Collective,
          attributes: [],
```